src/table.ts

```diff
@@ -482,6 +487,76 @@ Please use the format 'prezzy' or '${instance.name}/tables/prezzy'.`);
     };
   }
 
+  backup(
+    id: string,
+    fields: Required<ModifiableBackupFields>,
+    gaxOptions?: CallOptions
+  ): Promise<CreateBackupResponse>;
+  backup(
+    id: string,
+    fields: Required<ModifiableBackupFields>,
+    gaxOptions: CallOptions,
+    callback: CreateBackupCallback
+  ): void;
+  backup(
+    id: string,
+    fields: Required<ModifiableBackupFields>,
+    callback: CreateBackupCallback
+  ): void;
+  /**
+   * Backup a table with cluster auto selection.
+   *
+   * Backups of tables originate from a specific cluster. This is a helper
+   * around Cluster.createBackup that automatically selects the first ready
+   * cluster from which a backup can be performed.
+   *
+   * @param {string} id A unique ID for the backup.
+   * @param {ModifiableBackupFields} fields Fields to be specified.
+   * @param {BackupTimestamp} fields.expireTime When the backup will be
+   *   automatically deleted.
+   * @param {CallOptions | CreateBackupCallback} [gaxOptionsOrCallback]
+   * @param {CreateBackupCallback} [cb]
+   * @return {void | Promise<CreateBackupResponse>}
+   *
+   * @example <caption>include:samples/document-snippets/table.js</caption>
+   * region_tag:bigtable_create_table
+   */
+  backup(
```

Comment:
Not sure about this convenience function, as it internally  sends additional RPC.